src/app/devices/module/moduleIndentityDetail/state.ts

```diff
@@ -0,0 +1,18 @@
+/***********************************************************
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License
+ **********************************************************/
+import { Record } from 'immutable';
+import { IM } from '../../../shared/types/types';
+import { SynchronizationWrapper } from '../../../api/models/synchronizationWrapper';
+import { ModuleIdentity } from '../../../api/models/moduleIdentity';
+import { SynchronizationStatus } from '../../../api/models/synchronizationStatus';
+
+export interface ModuleIdentityDetailStateInterface extends SynchronizationWrapper<ModuleIdentity>{}
+
```

Comment:
I like the way you have your state interface defined. I went with composition not inheritence that created an uncessary level. Will snap to yours once your change gets in.